vars.py

```diff
@@ -117,6 +118,9 @@ def useLegacySoundtrack(self):
     def try_djose_skip(self):
         return self.attemptDjose
 
+    def get_force_blitz_win(self):
```

Comment:
Some day, we'll stop using gettters and setters for things that don't do anything than work like a pass through. But that day is not today.

Reply:
Just following the established pattern :)